src/bundle/Resources/public/scss/_icons.scss

```diff
@@ -95,3 +95,16 @@
         }
     }
 }
+
+.ez-preview__action {
+    .ez-icon {
+        width: 1rem;
```

Comment:
I think it's a good time to introduce icons mixins like: `icon-small`, `icon-medium`, `icon-big` that will define the icon size in a specific context. Or maybe, we predefine icons CSS classes instead.